bin/admin/twistedchecker-trunk-diff.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+#
+# Helper for running twistedchecker and reporting only errors that are part
```

Comment:
I have moved it in .travis, but I think that we are just fooling ourselves , as I will continue to use it from .travis directory as it better than nothing and better than doing a commit and push to see if my changes are introducing any errors :)